app/json/slate_lineup_configs.json

```diff
@@ -68,29 +50,13 @@
   },
   {
     "id": "b50524d6-4df9-4f15-a0d0-13ccc8bdf4ed",
-    "description": "Android Discover Tab Personalized Curator Topic",
+    "description": "Android Discover Tab Curator Topic",
     "experiments": [
       {
         "description": "default layout",
-        "rankers": [
-          "top1-topics"
-        ],
+        "rankers": [],
         "slates": [
-          "af2cc2c0-1286-47a3-b8ce-187b905f94af",
-          "00f0c593-3c99-4d5a-9297-c66f8b00be01",
-          "856eb5f8-da7f-43ae-ac5e-25da402af0ae",
-          "cceefa28-906e-47c5-b704-8c5b824ecd1b",
-          "651d27ab-a898-4173-9700-dd1726fbaaa4",
-          "951ae6c3-4438-458b-9936-7f3b8ff0f178",
-          "47023d21-0aa6-4f98-9077-eac21fad44f1",
-          "1a8de2be-db03-4a2f-901a-7c4c1cbd156a",
-          "f4b58337-4ab6-4563-9503-c384e773946f",
-          "f8c945b0-52fa-4203-bb31-6747245fe021",
           "b548b456-70c4-474d-a723-80d040d00fec",
```

Comment:
Kept science slate based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).